tests/jobs/test_delete_completed_applications.py

```diff
@@ -1,7 +1,10 @@
 import pytest
 from grc.models import db, Application, ApplicationStatus, ApplicationData
+from grc.utils.logger import LogLevel, Logger
 from unittest.mock import patch, MagicMock
 
+logger = Logger()
```

Comment:
This is not being used

Reply:
I assume you wanted to use it for line 30. Think that print statement can just be removed from the test